### ISSUE TITLE
Figure out behavior for these DiscardedSymbol APIs

### ISSUE THREAD

**Issue by jcouv** [dotnet/roslyn#15449]
In `DiscardedSymbol`:

```C#
        // PROTOTYPE(wildcards): need to figure out the correct behavior for the following methods.
        //public override string GetDocumentationCommentId() => TODO;
        //public override string GetDocumentationCommentXml(CultureInfo preferredCulture, bool expandIncludes, CancellationToken cancellationToken) => TODO;
    }
```

**Comment by gafter** [comment]
I believe they should return empty strings.

**Comment by gafter** [comment]
The current behavor is the proposed behavior. These comments are removed in #15548.

**Comment by gafter** [comment]
Fixed in #15548
